test/run-tests

```diff
@@ -185,10 +185,16 @@ class Task:
             # used to be tested by running all playbooks `test/test_*.yml`.
             # Support both, but prefer the standard way. Can be removed once
             # all repos are moved over.
-            playbooks = glob.glob(f'{sourcedir}/tests/tests*.yml')
+            playbookglob = f'{sourcedir}/tests/tests*.yml'
+            playbooks = glob.glob(playbookglob)
             if not playbooks:
                 playbooks = glob.glob(f'{sourcedir}/test/test_*.yml')
 
+            if not playbooks:
+                print(f'No test playbooks found, please add at least one'
+                      'playbook that matches {playbookglob}.')
```

Comment:
The problem is that some roles have no tests.

Reply:
Could be solved by excluding those (which don't have tests yet) from the list of repositories in `config.json`.